app/models/pac_bio_sample_prep_request.rb

```diff
@@ -26,14 +26,14 @@ def on_started
   end
```

Comment:
I'm aware of the slight change in behaviour. It allows the un-needed plate creation stage of legacy PacBio to be switched off without issue. The existing pipeline generate an RT ticket pretty much the same time as I was fixing the final method in this class, so I overhauled the whole thing.